src/rexdb.py

```diff
@@ -124,3 +128,76 @@ def get_data_at_range(self, start_time: time.struct_time, end_time: time.struct_
                 print(f"could not search file: {e}")
 
         return entries
+
+    def get_field_filtered(self,
```

Comment:
Another thought, given our memory issues we probably want to require a limit on the number of entries you get back.
Otherwise calling `get_field_filtered` is pretty dangerous, you have no idea how much memory will be allocated in the worst case.